SMLHelper/Patchers/LanguagePatcher.cs

```diff
@@ -75,11 +71,11 @@ private static void WriteOriginalLinesFile(string modKey)
             var text = new StringBuilder();
             foreach (string langLineKey in modCustomLines.Keys)
             {
-                string value = modCustomLines[langLineKey];
-                text.AppendLine($"{langLineKey}{KeyValueSeparator}{TextDelimiterOpen}{value}{TextDelimiterClose}");
+                string value = modCustomLines[langLineKey].Replace("\n", "\\n").Replace("\r", "\\r");
+                text.AppendLine($"{langLineKey}{KeyValueSeparator}{value}");
```

Comment:
Better backwards compatibility could be had if we maintain the old `TextDelimiterOpen`|`TextDelimiterClose` values.  
They can be trimmed safely by index by removing the first and last characters of the captured value after the `KeyValueSeparator`.

Reply:
They are completely useless. I could implement a conditional check to only remove them if they are present.

Reply:
A conditional check would actually not work. Stuff like `{0}CustomValue{1}` would become `0}CustomValue{1`. We either completely get rid of the text delimiters, or keep them forever. (There is literally no point to keep them except backwards compatibility, and we're talking about Unity 2018 here...)

Reply:
> There is literally no point to keep them except backwards compatibility

Precisely.  
While the Unity 2018 will cause a lot of headaches for many, users should only really need to update their mods as updates are made.  
I am perfectly aware that we no longer need delimiters after this update, but I am making the case that if we can save 99% of users the hassle of having to rebuild or update their override files, it's worth it.

You're welcome to make the case that this isn't necessary. Otherwise, I can add these updates in myself later tonight.

Reply:
What if a text delimiter is not found? Do we just throw an error or something? And refuse to load the line even if it's perfectly fine?

Reply:
Resolved without re-writing the original file.  
Entries that use `{`|`}` delimiters wrapping the value will be safely removed before overriding without accidentally removing string.format tokens.